package.json

```diff
@@ -62,6 +62,10 @@
 				"command": "rescript-vscode.stop_dead_code_analysis",
 				"title": "ReScript: Stop dead code analysis."
 			},
+			{
+				"command": "rescript-vscode.restartLanguageServer",
```

Comment:
the other commands use `snake_case`, so please let's stay consistent here and use `restart_language_server`.

Reply:
Thanks for the review @fhammerschmidt  . I have updated it to snake_case.